06_gpu_and_ml/comfyui/comfyapp.py

```diff
@@ -101,9 +101,7 @@ def hf_download():
 )
 
 # Lastly, we copy the ComfyUI workflow JSON to the container.
-image = image.add_local_file(
-    Path(__file__).parent / "workflow_api.json", "/root/workflow_api.json"
-)
```

Comment:
I think this was the first time a lot of examples were exposed to the upgraded `ruff`; sorry for the noise.